test/integration/i18n-support/test/index.test.js

```diff
@@ -412,6 +412,37 @@ describe('i18n Support', () => {
         }
       })
 
+      it('should return 404 error for repeating locales', async () => {
+        const defaultLocale = 'en-US'
+        for (const locale of nonDomainLocales) {
+          for (const asPath of [
+            '/gsp/fallback/always/',
+            '/post/comment/',
+            '/gssp/first/',
+          ]) {
+            const res = await fetchViaHTTP(
+              curCtx.appPort,
+              `/${locale}/${defaultLocale}${asPath}`,
+              undefined,
+              {
+                redirect: 'manual',
+              }
+            )
+            expect(res.status).toBe(404)
+            const $ = cheerio.load(await res.text())
+            const props = JSON.parse($('#props').text())
+            console.log(props)
```

Comment:
leftover console.log?

Reply:
Oh, sorry, my mistake. Added these lines at the last moment to check what it is. Next time, of course, I will recheck PRs better.
How I can fix it? Create new PR with fix?

Reply:
Created PR.
https://github.com/vercel/next.js/pull/31289

Reply:
console.logs are alright in tests as they can give more context when something fails so no need to worry